docs/slack.rst

```diff
@@ -174,54 +174,130 @@ Our goal with Write the Docs is to be **the most welcoming community on the inte
+  Don't make them feel bad; they are likely just in a different context.
 - **Let Me Google That For You (LMGTFY)** - If people ask questions that are simple to answer on Google, still answer them nicely.
-- **Question Policing** - Some people might ask incomplete questions that are hard to answer. Ask follow up questions and act in good faith to help them solve their issues.
-- **Technology Competitions** - It doesn't matter if your tool is faster or better in some metric. We care about creating great documentation for our users.
+- **Question Policing** - Some people might ask incomplete questions that are hard to answer.
+  Ask follow up questions and act in good faith to help them solve their issues.
+- **Technology Competitions** - It doesn't matter if your tool is faster or better in some metric.
+  We care about creating great documentation for our users.
+
+Posting rules
+~~~~~~~~~~~~~
+
+The Write the Docs Slack hosts many documentarians and has few moderators.
+We expect everyone to follow these "rules of the road" at all times, to keep the conversation going smoothly and to reduce annoyances for everyone.
+
+- **Start and stay on topic.**
+  Digressions happen, but don't be afraid to move a conversation to a thread or another channel when it makes sense.
+- **Focus on quality over quantity.**
+  Excessive crossposting or reposting can drown out other conversations and looks like spam.
+  Send your message to the most-relevant channel then give some time for conversations to start before casting a wider net.
+  Sometimes it makes sense to send a similar message to multiple channels at once, or to post again, but show restraint.
+- **Include context with links.**
+  Give us your perspective: tell us what you're sharing, why, and your connection to it.
+  Don't post a bare URL or use link shorteners: they're not informative and look like spam.
+- **Use content warnings and threads for sensitive topics.**
+  Write the Docs is a big community: topics that seem safe to you could make others feel uncomfortable or unwelcome.
+  Help others choose which conversations they opt into by signposting sensitive or traumatic topics and confining them to threads.
+
+Posts not meeting these criteria may be deleted.
+
+Self-promotion, surveys, and sales
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You're welcome to share, link to, and mention your own work, including blog posts, social media, events, and paid products or services. That said, posts where you stand to benefit from others' money, personal information, or labor, come with higher expectations.
+When this applies, make sure you follow the general guidelines closely.
+
+In addition to the guidance that applies to all participants, we expect self-promotion on the Write the Docs Slack to follow these additional guidelines:
+
+- **Post as yourself, not as a brand.**
+  Write the Docs Slack accounts must be operated by an individual human being that exists outside of work.
```

Comment:
You're right, this is a more general rule. I'll move the whole bullet to the posting rules section.

It might be nice to include some similar text in the signup form, actually, since that's more or less _when_ people are most likely going to "violate" the rule. It might be a good addition to the email field (e.g., "Don't use a shared inbox or alias. Write the Docs Slack accounts are for real people, not brands. An account must be operated by an individual human being.")

Reply:
Moved the bullet with bb77c987.

Reply:
Love the idea of adding it to the signup form. I actually don't know if we have control over that though..